public/js/chat.js

```diff
@@ -13,10 +13,50 @@ const locationTemplate = document.querySelector('#locationTemplate').innerHTML
 const sidebarTemplate = document.querySelector('#sidebarTemplate').innerHTML
 
 // Options
-const { username, room } = Qs.parse(location.search, {
+const { username, room, roomId } = Qs.parse(location.search, {
     ignoreQueryPrefix: true,
 })
 
+const roomMessagesRender = async (roomId) => {
+    const response = await fetch(`/messages?roomId=${roomId}`)
```

Comment:
the url doesn't follow REST

Reply:
Could you please clarify why?

Reply:
roomId should be passed in path, not in query. query part is used for searching/filtering

Reply:
Fixed.